lib/services/tankovna.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+var _ = require('lodash')
+var xray = require('x-ray')
+var moment = require('moment')
+
+module.exports = function () {
+  var x = xray()
+
+  var map = {
+    leden: '01',
+    únor: '02',
+    březen: '03',
+    duben: '04',
+    květen: '05',
+    červen: '06',
+    červenec: '07',
+    srpen: '08',
+    září: '09',
+    říjen: '10',
+    listopad: '11',
+    prosinec: '12'
+  }
+
+  var parseDate = function (dateString) {
+    return moment(dateString.replace(/^.+,\s+/, '').replace(' (dnes)', '').replace(new RegExp(_.keys(map).join('|')),
+      function (replacer) {
+        return map[replacer]
+      }), 'D M')
```

Comment:
pokud se tohle používá na více místech, tak to vytáhni do společného souboru alá lib/zomato